python/interpret_text/common/utils_unified.py

```diff
@@ -6,150 +6,17 @@
 # https://github.com/microsoft/nlp-recipes/tree/master/utils_nlp
 
 """
-    Utility functions for downloading, extracting, and reading the
-    Multi-Genre NLI (MultiNLI) Corpus.
-    https://www.nyu.edu/projects/bowman/multinli/
+    Utility functions useful across explainers
 """
-
-import os
-
-import pandas as pd
 import logging
 
-from tempfile import TemporaryDirectory
-
 import torch
 from pytorch_pretrained_bert import BertTokenizer
```

Comment:
We should have a github issue to remove pytorch_pretrained_bert from the dependencies